frontend/src/metabase/sharing/components/PublicLinkPopover/PublicLinkPopover.tsx

```diff
@@ -62,7 +62,13 @@ export const PublicLinkPopover = ({
       position="bottom-end"
     >
       <Popover.Target>
-        <Box onClick={isOpen ? onClose : undefined}>{target}</Box>
+        <Box
+          pos="absolute"
+          style={{ right: 0 }}
```

Comment:
nit: Mantine's style props also has `right`

```suggestion
          right={0}
```